### PR TITLE
WIP BXMSDOC-5410: New Section "Creating a project using Archetypes" for Managing Projects doc

### PR DESCRIPTION
**Pull request by hmanwani-rh**
See JIRA: https://issues.redhat.com/browse/BXMSDOC-5410

[RHPAM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-5410-RHPAM/#create_archetype_project)
[RHDM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-5410-RHDM/#create_archetype_project)

### CODE REVIEW

**Review by caponetto — APPROVED** [review]

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[id='create_archetype_project']
+= Creating a project using Archetypes
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: Archetypes lower case

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Archetypes are projects that are installed in maven repositories in which a template structure is set or can be created if required. You can use archetypes to create a project in {CENTRAL}. When a project is created in {CENTRAL} it is added to the Git repository that is connected to your {PRODUCT} installation.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: Maven-capitalize and use Apache Maven on first use.

passive voice:
or can be created if required.

I'm a little confused by this:
in which a template structure is set or can be created if required.
Is there a special Maven repo that lets us create template structures or can we do that in all Maven repos? How do I create a template structure? Or is it the archetype that creates the structure?

When you create a project, it is added...

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+.Prerequisites
+
+* You have created an archetype and listed in the {CENTRAL} *Settings* from the maven repository.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: list it in the ...?

This is confusing. I don't have the UI handy. Do you mean a Maven repository section in Settings?
 *Settings* from the maven repository.

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+* You have set an archetype as default in your space in the {CENTRAL}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: You have set a default archetype in your space in {CENTRAL} (no 'the' before)
So you have these prereqs, but I don't see instructions in Managing Projects for how to create or set an archetype as default. If it's in another book add an xref to that book. If not, consider adding it here.

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
+. Click *Add Project*.
+. Type the project name and description in the *Name* and *Description* field respectively.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: fields

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+. Click *Configure Advanced Options*.
+. Enable *Based on template* checkbox.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: Select the *Based....

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+. Select the archetype from drop-down options if required. The default archetype is selected that is already set in the space.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc:20]: What do I actually click for this. Is there a menu? Does it have a name?
If you want to use an archtetype other than the default archetype, select an archetype from the *BLAH* menu.

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+. Click *Add*.
+
+The *Assets* view of the project opens when you successfully create the project.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: The *Assets* view of the project opens. (delete the rest)
If that is what happens after you click Add.

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc (hunk 4 above)
Comment (hmanwani-rh) [reply doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: @emmurphy1, how to create and add archetype will be covered in Configuring BC settings and properties doc and conditionalized, once completed.

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc (hunk 7 above)
Comment (hmanwani-rh) [reply doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc:20]: @emmurphy1, when the `Based on template` is enabled, then a drop-down (No-name given) below becomes active showing the default archetype as selected.

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc (hunk 1 above)
Comment (hmanwani-rh) [reply doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: @emmurphy1, what if UI consists of capitalized Archetypes?

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 9]
```diff
@@ -0,0 +1,21 @@
+[id='create_archetype_project']
+= Creating a project using archetypes
+
+Archetypes are projects that are installed in Apache Maven repositories and contain a specific template structure. You can also generate parameterized versions of the project templates using archetypes. You can use archetypes to create a project in {CENTRAL}. When you create a project, it is added to the Git repository that is connected to your {PRODUCT} installation.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: Remove this:
You can use archetypes to create a project in {CENTRAL}.

Change this:
When you use an archetype to create a project, it is added to the Git repository that is connected to your {PRODUCT} installation.

Note that my version might not be 100 percent correct - I'm not familiar with the feature.

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+.Prerequisites
+
+* You have created an archetype and listed in the {CENTRAL} *Settings*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: I would be  more specific. You have created an archetype and added it to the Archetype section of the {CENTRAL} *Settings* page. 

BTW, do I create an archetype in BC or Maven (I really don't know). Mabye add a link;
For information about creating archetypes, see ...

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+* You have set a default archetype in your space in {CENTRAL}.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc:9]: Maybe an xref for how to do this?

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects*.
+. Click *Add Project*.
+. Type the project name and description in the *Name* and *Description* fields respectively.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: Enter a project name.... *Description* fields. Remove respectively. Not necessary.

File: doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+. Click *Configure Advanced Options*.
+. Select the *Based on template* checkbox.
+. If you want to use an archetype other than the default archetype, select an archetype from the drop-down options.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: Not sure about 'drop-down options'. Do you click the existing archetype name to see a list of other archetypes? If the  drop-down doesn't have a name you need to describe it something like I have.

Comment (hmanwani-rh) [reply doc-content/enterprise-only/project-data/create-archetype-project-proc.adoc]: how about changing to this "If you want to use an archetype other than default archetype, click on the default archetype and select an archetype from the drop-down."

**Pull request merged**